src/api/app/assets/stylesheets/webui2/bootstrap_variables/tables.scss

```diff
@@ -1,2 +1,6 @@
 // For better contrast in tables using .table-striped
-$table-accent-bg: $gray-300;
+$table-accent-bg: $gray-100;
+
+table.dataTable tbody tr:hover {
```

Comment:
Why not [`.table-hover`](https://getbootstrap.com/docs/4.3/content/tables/#hoverable-rows)?